### DISCUSSION TITLE
Space character in URL

### DISCUSSION THREAD

**Discussion by osorio** [WinMerge/winmerge#1518] [Q&A]
How do I make WinMerge recognize a space character in a file and folder name, when called from the command line?

**Comment by sdottaka** [comment]
Please enclose the path in double quotation marks
~~~
"C:\Program Files\WinMerge\WinMergeU.exe" "C:\Program Files\WinMerge\Docs\" "D:\dev\winmerge\Docs\"
~~~

**Comment by Gitoffthelawn** [comment]
@osorio Just FYI, you will find this to be true for almost all Windows CLI applications.  Any path with a space requires the path to be enclosed in double-quotes.  Because it's true for the Windows CLI, it's also true when writing batch files.

**Reply by osorio** [reply]
Thanks both!

**Reply by Gitoffthelawn** [reply]
You're welcome!
